commit 918650731803579659fa6b5eedf3f80310afc59f
Author: Paul Eggert <eggert@cs.ucla.edu>
Date:   Thu Oct 27 09:51:13 2022 -0700

    Fiji no longer observes DST
    
    * NEWS: Mention this (thanks to Shalvin Narayan).
    * australasia (Fiji): Remove our incorrect guess for spring 2022
    on.  For now, guess no DST indefinitely.

diff --git a/NEWS b/NEWS
index 546fe114..2c905fda 100644
--- a/NEWS
+++ b/NEWS
@@ -4,6 +4,9 @@ Unreleased, experimental changes
 
   Changes to future timestamps
 
+    Fiji will not observe DST in 2022/3.  (Thanks to Shalvin Narayan.)
+    For now, assume DST is suspended indefinitely.
+
     Jordan and Syria are abandoning the DST regime and are changing to
     permanent +03, so they will not fall back from +03 to +02 on
     2022-10-28.  (Thanks to Steffen Thorsen and Issam Al-Zuwairi.)
diff --git a/australasia b/australasia
index df5a52f0..af0410ab 100644
--- a/australasia
+++ b/australasia
@@ -386,8 +386,14 @@ Zone Antarctica/Macquarie 0	-	-00	1899 Nov
 # concerned shifting arrival and departure times, which may look like a simple
 # thing but requires some significant logistical adjustments domestically and
 # internationally."
-# Assume for now that DST will resume with the recent pre-2020 rules for the
-# 2022/2023 season.
+
+# From Shalvin Narayan (2022-10-27):
+# Please note that there will not be any daylight savings time change
+# in Fiji for 2022-2023....
+# https://www.facebook.com/FijianGovernment/posts/pfbid0mmWVTYmTibn66ybpFda75pDcf34SSpoSaskJW5gXwaKo5Sgc7273Q4fXWc6kQV6Hl
+#
+# From Paul Eggert (2022-10-27):
+# For now, assume DST is suspended indefinitely.
 
 # Rule	NAME	FROM	TO	-	IN	ON	AT	SAVE	LETTER/S
 Rule	Fiji	1998	1999	-	Nov	Sun>=1	2:00	1:00	-
@@ -402,8 +408,6 @@ Rule	Fiji	2014	2018	-	Nov	Sun>=1	2:00	1:00	-
 Rule	Fiji	2015	2021	-	Jan	Sun>=12	3:00	0	-
 Rule	Fiji	2019	only	-	Nov	Sun>=8	2:00	1:00	-
 Rule	Fiji	2020	only	-	Dec	20	2:00	1:00	-
-Rule	Fiji	2022	max	-	Nov	Sun>=8	2:00	1:00	-
-Rule	Fiji	2023	max	-	Jan	Sun>=12	3:00	0	-
 # Zone	NAME		STDOFF	RULES	FORMAT	[UNTIL]
 Zone	Pacific/Fiji	11:55:44 -	LMT	1915 Oct 26 # Suva
 			12:00	Fiji	+12/+13
